src/main/java/by/onliner/recorder/VideoManager.java

```diff
@@ -0,0 +1,51 @@
+package by.onliner.recorder;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.Collections;
+import java.util.LinkedList;
+import java.util.List;
+
+public class VideoManager {
+
+    private static final ThreadLocal<VideoRecorder> threadRecorder = new ThreadLocal<>();
+    private static final List<VideoRecorder> recorders = Collections.synchronizedList(new LinkedList<>());
+
+    protected static final Logger logger = LogManager.getLogger(VideoManager.class);
+
+    public static VideoRecorder getVideoRecorder() {
+        if (null == threadRecorder.get()) {
```

Comment:
Why do you compare null with recorder not recorder with null?

Reply:
Fixed